client-app/src/components/UnProtectedRout.jsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { Navigate, Outlet } from "react-router-dom";
+
+function UnProtectedRout({ authorized }) {
+  return !authorized ? <Outlet /> : <Navigate to="/" />;
```

Comment:
If the user is already authorized, won't this cause a recursion?
`return !authorized ? <Outlet /> : <Navigate to="/" />;`
Should it be like this?
`return !authorized ? <Outlet /> : <Navigate to="/home" />;`